### PR TITLE
fixed small error

### PR DESCRIPTION
**Pull request by 0xflotus**

### CODE REVIEW

File: photon/use-only-photon.md [hunk 1]
```diff
@@ -130,7 +130,7 @@ Once provided, the CLI will prompt you to select one of the existing **databases
 
 <Details><Summary>Database credentials for <strong>PostgreSQL</strong></Summary>
 <br />
-When using PostgreSQL, you need to provide the following infomration to connect your existing MySQL database server:
+When using PostgreSQL, you need to provide the following information to connect your existing MySQL database server:
 
 - **Host**: The IP address/domain of your database server, e.g. `localhost`.
 - **Post**: The port on which your database server listens, e.g. `5432` (PostgreSQL) or `3306` (MySQL).
```
Comment (chill5018) [inline photon/use-only-photon.md]: Should **Post** be **Port**? If so, there are multiple of these that could be corrected.

Comment (0xflotus) [reply photon/use-only-photon.md]: I think you're right.

**Comment by nikolasburk** [comment]
Thanks so much for catching all of this and being my spellchecker @chill5018 😄🙏

**Comment by pantharshit00** [comment]
Hey @0xflotus 

Would you mind signing our CLA for this project? Since our repos are new, we didn't have cla assistant setup at that time. 

Please sign it here: https://cla-assistant.io/prisma/prisma2-docs?pullRequest=1


If you don't sign this, we will, unfortunately, have to remove your commit.

**Comment by nilubava** [comment]
Fixed in https://app.netlify.com/sites/prisma2/deploys/5e78d021608bbe0007674b72

**Pull request merged**